Add CardinalMobile.xcframework version 2.2.5-1 (#649)
braintree/braintree_ios: SWIFT_PACKAGE_MANAGER.md

### BraintreeThreeDSecure

> ~Currently, to use the `BraintreeThreeDSecure` library via SPM, you must manually include `CardinalMobile.framework`. Once we receive an xcframework version of CardinalMobile, these steps will no longer be required.~ These steps are only required for versions 5.0.0 to 5.2.0. For versions _greater than_ 5.2.0, you can include CardinalMobile via Swift Package Manager.

Reviewer: I feel like crossing this statement out is confusing since above we are directing people to this section if they are using version 5.0.0-5.2.0

Author: Hmmm... I can rework this section.

Author: @sarahkoop Updated! Could you take a look and let me know if my changes make sense?

Reviewer: I think this is very clear now 👍

Author: Awesome, thanks for the feedback!

> ~Currently, to use the `BraintreeThreeDSecure` library via SPM, you must manually include `CardinalMobile.framework`. Once we receive an xcframework version of CardinalMobile, these steps will no longer be required.~ These steps are only required for versions 5.0.0 to 5.2.0. For versions _greater than_ 5.2.0, you can include CardinalMobile via Swift Package Manager.

Author: ```suggestion
If you are using versions 5.0.0 to 5.2.0, in order to use the `BraintreeThreeDSecure` library with SPM, you must manually include `CardinalMobile.framework`. For versions _greater than_ 5.2.0, you can include CardinalMobile via Swift Package Manager.
```
